msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -214,6 +224,7 @@ The meetings app capabilities are declared in your app manifest via the **config
 > * Please use [Developer Preview manifest schema](../resources/schema/manifest-schema-dev-preview.md) to try this in your app manifest.
```

Comment:
Suggest removing the bullet since it is a single sentence.
```Suggestion
> * Please use [Developer Preview manifest schema](../resources/schema/manifest-schema-dev-preview.md) to try this in your app manifest.
```